contracts/L2_TokenRetriever.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.19;
+
+import "./Lockable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import "@uma/core/contracts/common/implementation/MultiCaller.sol";
```

Comment:
Do you still want this import?

Reply:
Removed https://github.com/across-protocol/contracts/pull/605/commits/d3f624bff2d3b263be3a310a5c9ef36a19075bc0